### PR TITLE
Updated persistent subscriptions documentation to include latest samples from gRPC clients releases

### PR DESCRIPTION
**Pull request by oskardudycz**

### CODE REVIEW

File: import/repos.json [hunk 1]
```diff
@@ -81,8 +81,8 @@
     "repo": "https://github.com/EventStore/EventStoreDB-Client-Java",
     "branches": [
       {
-        "version": "1.0.0",
-        "name": "trunk"
+        "version": "2.0.0",
+        "name": "persistent_subscriptions_samples"
```
Comment (oskardudycz) [inline import/repos.json]: **TODO:** Update it once https://github.com/EventStore/EventStoreDB-Client-Java/pull/133 is merged.

File: docs/clients/grpc/persistent-subscriptions.md [hunk 2]
```diff
@@ -19,7 +38,7 @@ Once you have created a subscription group, clients can connect to that subscrip
 
 The most important parameter to pass when connecting is the buffer size. This represents how many outstanding messages the server should allow this client. If this number is too small, your subscription will spend much of its time idle as it waits for an acknowledgment to come back from the client. If it's too big, you waste resources and can start causing time out messages depending on the speed of your processing.
 
-@[code{subscribe-to-persistent-subscription-to-stream}](../dotnet/21.2/samples/persistent-subscriptions/Program.cs)
+@[code{subscribe-to-persistent-subscription-to-stream}](../dotnet/21.6/samples/persistent-subscriptions/Program.cs)
```
Comment (ylorph) [inline docs/clients/grpc/persistent-subscriptions.md]: @oskardudycz do we have sample for this in the other langs ?

Comment (alexeyzimarev) [reply docs/clients/grpc/persistent-subscriptions.md]: Also, let me check if it is replaceable with an alias.

Comment (oskardudycz) [reply docs/clients/grpc/persistent-subscriptions.md]: I forgot to update that. It's done now using an alias.

File: import/repos.json [hunk 3]
```diff
@@ -124,8 +124,8 @@
     "repo": "https://github.com/EventStore/EventStore-Client-Go.git",
     "branches": [
       {
-        "version": "1.0.0",
-        "name": "master"
+        "version": "1.0.2",
+        "name": "persistent_subscriptions_snippets_naming"
```
Comment (oskardudycz) [inline import/repos.json]: **TODO:** Update it once https://github.com/EventStore/EventStore-Client-Go/pull/97 is merged.

**Review by ylorph — APPROVED** [review]

**Review by alexeyzimarev — APPROVED** [review]

**Pull request merged**